external-import/sentinel-incidents/src/sentinel_incidents_connector/utils.py

```diff
@@ -48,3 +49,24 @@ def is_ipv4(value: str) -> bool:
         return True
     except ipaddress.AddressValueError:
         return False
+
+
+def find_matching_file_id(malware_name: str, stix_objects: list) -> list | None:
```

Comment:
I think the name of this function doesn't really fit it's behavior

Reply:
Yes, you're right again, thank you for your comment I've corrected that ;)